### PR TITLE
add instructions for running performance tests

### PR DESCRIPTION
**Pull request by aelkiss**
- instructions for binary resource test
- fix typo in property name for file sizes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,3 +18,52 @@ The following is an example running a test against a remote Fedora deployed unde
...
+RandomStringUtils implementation can generate about 100MB/second of random
+data on a i7 3770.
+
+(Ideally instead of generating the entire file at once, we could generate small
+chunks at a time and just have the sampler read them from an InputStream, but
+that doesn't seem to be possible with the default JMeter HTTP sampler.)
+
+To run with a maximum file size of 500MB:
+
+* Toggle thread groups so only 'Fedora4 Create Binary Resource' is enabled (can only be done via the GUI, but persists after saving)
```
Comment (escowles) [inline README.md]: I found enabling/disabling thread groups in the GUI was reflected pretty directly in the `fedora.jmx` file, e.g.:

```
160c160
<       <ThreadGroup guiclass="ThreadGroupGui" testclass="ThreadGroup" testname="Fedora4 Create New Containers" enabled="false">
---
>       <ThreadGroup guiclass="ThreadGroupGui" testclass="ThreadGroup" testname="Fedora4 Create New Containers" enabled="true">
```

Can we use variables to enable/disable ThreadGroups with command-line parameters?  That would make it easier for people to run the different tests.

**Comment by aelkiss** [comment]
This article has some suggestions on how to do this: http://www.testautomationguru.com/jmeter-manage-test-plan/ - essentially one needs to set the number of threads for thread groups you don't want to run to zero. I can see about adding appropriate properties to do that later this week.

**Comment by escowles** [comment]
The size parameters for tests 1 and 2 don't seem to be working.  Regardless of the parameters I use, I'm seeing 0-byte files created in all cases.

**Update**: ignore this comment -- looks like my testing script was using the wrong version of the `fedora.jmx` file.  Now that I'm using the correct test script, I'm seeing file sizes in the specified range.  So I'll restart tests 1 and 2 on Princeton's VMs, and hopefully have some results for Monday's meeting.

File: README.md (hunk 1 above)
Comment (awoods) [inline README.md]: :+1: to toggling ThreadGroups via CLI parameters.

**Comment by aelkiss** [comment]
Sorry I didn't get to this last week; PR is updated just in time for the call!

**Pull request merged**